### PR TITLE
:book: improve docs for mailbox controller

### PR DESCRIPTION
**Pull request by pdettori**
## Summary

This PR improves the documentation for trying out the mailbox controller.

### CODE REVIEW

File: docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md [hunk 1]
```diff
@@ -41,7 +41,10 @@ The command line flags, beyond the basics, are as follows.
 
 To exercise it, do the following steps.
 
-Start a kcp server.  Do the remaining steps in a separate shell, with
+Clone this repo, install kcp (v0.11.0 is suggested) and start a kcp server
```
Comment (MikeSpreitzer) [inline docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md:44]: s/v0.11.0 is suggested/the version in `go.mod` is required/

File: docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md [hunk 2]
```diff
@@ -105,26 +111,37 @@ That should provoke logging like the following from the mailbox controller.
...
+And you can verify that as follows:
 
 ```console
-$ kubectl ws root:edge
-Current workspace is "root:edge".
+$ kubectl ws root:espw
+Current workspace is "root:espw".
 
-$ kubectl get Workspace
+$ kubectl get workspace
```
Comment (MikeSpreitzer) [inline docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md:120]: `Workspace` is the "kind", `workspaces` is the "resource"; `workspace` is nothing but a convenience for sloppiness, not something we should recommend.

File: docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md [hunk 3]
```diff
@@ -105,26 +111,37 @@ That should provoke logging like the following from the mailbox controller.
...
+Next, `kubectl delete` that workspace, and watch the mailbox
 controller wait for it to be gone and then re-create it.
 
 ```console
 I0305 18:08:15.428884   85556 main.go:369] "Created missing workspace" worker=2 mbwsName="niqdko2g2pwoadfb-mb-f99e773f-3db2-439e-8054-827c4ac55368"
 ```
 
-Finally, go back to your inventory workspace and `kubectl delete
-SyncTarget stest1` and watch the mailbox controller react as follows.
+Finally, go back to your inventory workspace:
```
Comment (MikeSpreitzer) [inline docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md:132]: How about retaining the English description of what is about to be done?

File: docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md (hunk 2 above)
Comment (pdettori) [reply docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md]: Thanks, I meant to use the resource (`workspaces`) - fixed in latest push.

File: docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md (hunk 3 above)
Comment (pdettori) [reply docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md]: addressed with latest push, thx

**Pull request merged**